lib/runner.js

```diff
@@ -20,6 +23,7 @@ var chainableMethods = {
 		before: {type: 'before'},
 		after: {type: 'after'},
 		skip: {skipped: true},
+		todo: {skipped: true, todo: true},
```

Comment:
> I'm not even sure whether they should be counted as tests now!

I agree with that. But for the sake of simplicity, we can just mark them as skipped tests internally, to prevent any kind of execution.